src/other/resources/UIMessages/UIMessages.properties

```diff
@@ -146,3 +146,20 @@ websockets_message_field_payload = Payload
 websockets_message_title = WebSockets Message
 websockets_replay = Replay
 websockets_tool = WebSockets
+
+togglescript_tool = Toggle Script
+togglescript_select = Select
+togglescript_script = Script
+togglescript_no_scripts_found = No Suitable Scripts Found
+togglescript_httpfuzzerprocessor = Fuzzer HTTP Processor
```

Comment:
It would be better to obtain the names directly from ZAP (e.g. `script/view/listTypes/`) to avoid duplication (and it will ensure that new script types are properly shown).

Reply:
Thanks for the info, didn't realize the listTypes API also gave the UI name